cluster/charts/crossplane/values.yaml

```diff
@@ -1,213 +1,218 @@
+# -- The deployment strategy for the Crossplane and RBAC Manager pods.
 deploymentStrategy: RollingUpdate
 
 image:
-  # -- Crossplane image.
+  # -- Repository for the Crossplane pod image.
   repository: crossplane/crossplane
-  # -- Crossplane image tag: if not set, appVersion field from Chart.yaml is used.
+  # -- The Crossplane image tag. Defaults to the value of `appVersion` in Chart.yaml.
   tag: ""
-  # -- Crossplane image pull policy used in all containers.
+  # -- The image pull policy used for Crossplane and RBAC Manager pods.
   pullPolicy: IfNotPresent
 
-# -- Enable nodeSelector for Crossplane pod.
+# -- Add `nodeSelectors` to the Crossplane pod deployment.
 nodeSelector: {}
-# -- Enable tolerations for Crossplane pod.
+# -- Add `tolerations` to the Crossplane pod deployment.
 tolerations: []
-# -- Enable affinity for Crossplane pod.
+# -- Add `affinities` to the Crossplane pod deployment.
 affinity: {}
 
-# -- Enable hostNetwork for Crossplane. Caution: setting it to true means Crossplane's Pod will have high privileges.
+# -- Enable `hostNetwork` for the Crossplane deployment. Caution: enabling `hostNetwork`` grants the Crossplane Pod access to the host network namespace.
 hostNetwork: false
 
-# -- Custom labels to add into metadata.
+# -- Add custom `labels` to the Crossplane pod deployment.
 customLabels: {}
 
-# -- Custom annotations to add to the Crossplane deployment and pod.
+# -- Add custom `annotations` to the Crossplane pod deployment.
 customAnnotations: {}
 
 serviceAccount:
-  # -- Custom annotations to add to the serviceaccount of Crossplane.
+  # -- Add custom `annotations` to the Crossplane ServiceAccount.
   customAnnotations: {}
 
-# -- Enable leader election for Crossplane Managers pod.
+# -- Enable leader election for the Crossplane pod.
 leaderElection: true
-# -- A list of additional args to be passed to Crossplane's container.
+# -- Add custom arguments to the Crossplane pod.
 args: []
 
 provider:
-  # -- The list of Provider packages to install together with Crossplane.
+  # -- A list of Provider packages to install.
   packages: []
 
 configuration:
-  # -- The list of Configuration packages to install together with Crossplane.
+  # -- A list of Configuration packages to install.
   packages: []
 
-# -- Names of image pull secrets to use.
+# -- The imagePullSecret names to add to the Crossplane ServiceAccount.
 imagePullSecrets: {}
 
 registryCaBundleConfig:
-  # -- Name of ConfigMap containing additional CA bundle for fetching from package registries.
-  name: {}
-  # -- Key to use from ConfigMap containing additional CA bundle for fetching from package registries.
-  key: {}
+  # -- The ConfigMap name containing a custom CA bundle to enable fetching packages from registries with unknown or untrusted certificates.
+  name: ""
+  # -- The ConfigMap key containing a custom CA bundle to enable fetching packages from registries with unknown or untrusted certificates.
+  key: ""
```

Comment:
Reminder to myself: I have to check how these were used and that there wasn't a specific reason for this to be a `{}`, happy to have it have the right type.

Reply:
All good 🎉